Jenkinsfile

```diff
@@ -698,7 +698,7 @@ spec:
                     }
 
                     // we use GKE for testing the generic platform
-                    def genericKversions = ["1.14"]
+                    def genericKversions = ["1.15"]
```

Comment:
What do you think of having several versions in the generic one?
For me sounds like `1.14`, `1.15`, `1.16` should be here

Reply:
> What do you think of having several versions in the generic one?
> For me sounds like `1.14`, `1.15`, `1.16` should be here

`1.16` not possible (uses GKE which doesn't yet have it available), re: `1.14` as this feature is experimental I wouldn't waste time in older kubernetes versions.